tests/ftests/078-sudo-cgcreate_systemd_scope.py

```diff
@@ -7,9 +7,10 @@
 # Author: Tom Hromatka <tom.hromatka@oracle.com>
 #
 
-from cgroup import Cgroup
 from process import Process
+from systemd import Systemd
 from libcgroup import Mode
+from cgroup import Cgroup
```

Comment:
Do you need this import?  I don't see any diffs that require it

Reply:
Yeah, it is not required by this change. It was more of a cosmetic change, to have a reverse xmas tree style